Introduce a 23video slide-type

danquah opened the pull request:
OSD-62

tripox commented on src/twentythree/Resources/public/templates/twentythree/slides/twentythree/twentythree.html:
```diff
@@ -0,0 +1,7 @@
+<div class="twentythree" style="position: relative">
```
Seems duplicate - already in the styling.

tripox commented on src/twentythree/Resources/public/templates/twentythree/slides/twentythree/twentythree.html:
```diff
@@ -0,0 +1,7 @@
+<div class="twentythree" style="position: relative">
+    <!-- https://www.twentythree.net/help/embedding-your-videos -->
+    <div style="width:100%; height:100%; position: relative;">
```
Consider putting the inline styling in the stylesheet if possible

tripox commented on src/twentythree/Resources/public/templates/twentythree/slides/twentythree/twentythree.js:
```diff
@@ -0,0 +1,50 @@
+                slide.url += "&hideBigPlay=1";
+                slide.url += "&showTray=0";
+                slide.url += "&defaultQuality=fullhd";
+                if (slide.options.sound === 1) {
```
Should this not be 0? 
The checkbox is enabling sound, not disabling.

tripox commented on src/twentythree/Resources/public/templates/twentythree/slides/twentythree/twentythree.js:
```diff
@@ -0,0 +1,50 @@
+            // Wait fadeTime before start to account for fade in.
+            region.$timeout(function () {
+                // Build the url for the video.
+                slide.url = "//video.kk.dk/v.ihtml/player.html?source=embed&photo%5fid=" + slide.options.id + "&autoPlay=1&loop=1";
```
Seems very specific for kk.dk?
Shouldn't it be configurable?

danquah committed:
Add a bundle for twentythree video

The bundle includes a tool that is used to configure the slide, and an advanced
integration that controls an embedded player.

OSD-62

danquah committed:
Making autoplay and the progressbar a bit more robust.

OSD-62

danquah commented:
@tripox did a fair bit of rework and have (amongst other things) addressed most of your comments. Could I have a re-review pretty please? :)
